include/nbl/builtin/glsl/ext/CentralLimitBoxBlur/default_compute_blur.comp

```diff
@@ -0,0 +1,108 @@
+#ifndef _NBL_GLSL_EXT_BLUR_INPUT_SET_DEFINED_
+#define _NBL_GLSL_EXT_BLUR_INPUT_SET_DEFINED_ 0
+#endif
+
+#ifndef _NBL_GLSL_EXT_BLUR_INPUT_BINDING_DEFINED_
+#define _NBL_GLSL_EXT_BLUR_INPUT_BINDING_DEFINED_ 0
+#endif
+
+#ifndef _NBL_GLSL_EXT_BLUR_OUTPUT_SET_DEFINED_
+#define _NBL_GLSL_EXT_BLUR_OUTPUT_SET_DEFINED_ 0
+#endif
+
+#ifndef _NBL_GLSL_EXT_BLUR_OUTPUT_BINDING_DEFINED_
+#define _NBL_GLSL_EXT_BLUR_OUTPUT_BINDING_DEFINED_ 1
+#endif
+
+#ifndef _NBL_GLSL_EXT_BLUR_INPUT_DESCRIPTOR_DEFINED_
+#define _NBL_GLSL_EXT_BLUR_INPUT_DESCRIPTOR_DEFINED_
+
+layout (set = _NBL_GLSL_EXT_BLUR_INPUT_SET_DEFINED_, binding = _NBL_GLSL_EXT_BLUR_INPUT_BINDING_DEFINED_, std430) restrict readonly buffer InputBuffer
+{
+	float in_values[];
+};
+
+#endif
+
+#ifndef _NBL_GLSL_EXT_BLUR_OUTPUT_DESCRIPTOR_DEFINED_
+#define _NBL_GLSL_EXT_BLUR_OUTPUT_DESCRIPTOR_DEFINED_
+
+layout (set = _NBL_GLSL_EXT_BLUR_OUTPUT_SET_DEFINED_, binding = _NBL_GLSL_EXT_BLUR_OUTPUT_BINDING_DEFINED_, std430) restrict writeonly buffer OutputBuffer
+{
+	float out_values[];
+};
```

Comment:
Do we really want the default input and output descriptors to be `float` SSBOs?

Reply:
well, summed area tables require LOTS of precision, also the only way to use 16bit types would be as STBs (imageBuffer) of R16_UNORM (16bit types in SSBOs require non-ubiquitous extensions).

Realistically speaking, whenever you use a 8 bit format, your LUT would need to be 16 or 32 bit, and whenever you use 16bit your LUT really needs to be 32bit

In fact `float` isnt that great, a "software" 32bit UNORM would have been better but we would have had to store a `float` denormalization constant per row.

What we should really be asking ourselves is "should we be giving `double` or `uint64_t` unorm as a intermediate storage option" ?

P.S. you can write this down as a comment, but please dont do anything about it now.